### PR TITLE
Log the render time

### PR DESCRIPTION
**Pull request by rogierslag**
This will include the render time in the `images` table. That way it will allow us to purge any caches which were rendered incorrectly due to a bug.

Flyby: Additionally fix an async/await which missed the await

@wouter-willems ready for review!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "iaas",
-  "version": "0.1.0",
+  "version": "0.1.3",
```
Comment (wouter-willems) [inline package.json]: what happened to the other 2 versions?

File: src/index.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ const uploadImage = async(req, res) => {
   const name = req.params.name;
   log('info', `Requested image upload for image_id ${name} with token ${sentToken}`);
 
-  const canConsumeToken = token(db).consume(sentToken, name);
+  const canConsumeToken = await token(db).consume(sentToken, name);
```
Comment (wouter-willems) [inline src/index.js]: why is this included?

File: package.json (hunk 1 above)
Comment (rogierslag) [inline package.json]: I forgot to tag those
